src/api/json/catalog.json

```diff
@@ -5138,6 +5138,18 @@
       ],
       "url": "https://raw.githubusercontent.com/synadia-io/connect/main/schemas/component.json"
     },
+    {
+      "name": "Tach",
+      "description": "Tach configuration file",
+      "fileMatch": ["tach.toml"],
+      "url": "https://raw.githubusercontent.com/gauge-sh/tach/refs/heads/main/public/tach-toml-schema.json"
+    },
+    {
+      "name": "Tach Domain",
+      "description": "Tach domain configuration file",
+      "fileMatch": ["tach.domain.toml"],
```

Comment:
I wasn't able to find documentation of how this is used, and I noticed that only some of the other entries specify globs like `**/config.toml`. If I expect this file to be found below the root of a project open in VSCode for example, do I need to use the glob pattern as well?

Reply:
You don't need to specify globs if you want to match on the exact filename